### PR TITLE
support custom content type viewers

### PR DESCRIPTION
**Pull request by geor-kasapidi**
For example - application/protobuf

### CODE REVIEW

File: Classes/Manager/FLEXManager+Private.h [hunk 1]
```diff
@@ -13,4 +13,7 @@
 /// An array of FLEXGlobalsTableViewControllerEntry objects that have been registered by the user.
 @property (nonatomic, readonly, strong) NSArray *userGlobalEntries;
 
+/// An array of FLEXCustomContentTypeViewer objects that have been registered by the user.
+@property (nonatomic, readonly, strong) NSMutableArray *customContentTypeViewers;
+
```
Comment (NSExceptional) [inline Classes/Manager/FLEXManager+Private.h]: `customContentTypeViewers` should be exposed as `NSArray` even in `FLEXManager+Private.h`

File: Classes/Network/FLEXNetworkTransactionDetailTableViewController.m [hunk 2]
```diff
@@ -250,7 +252,7 @@ + (FLEXNetworkDetailSection *)generalSectionForTransaction:(FLEXNetworkTransacti
         FLEXNetworkDetailRow *postBodySizeRow = [[FLEXNetworkDetailRow alloc] init];
         postBodySizeRow.title = @"Request Body Size";
         postBodySizeRow.detailText = [NSByteCountFormatter stringFromByteCount:[transaction.cachedRequestBody length] countStyle:NSByteCountFormatterCountStyleBinary];
-        [rows addObject:postBodySizeRow];
+        [rows addObject:  postBodySizeRow];
```
Comment (NSExceptional) [inline Classes/Network/FLEXNetworkTransactionDetailTableViewController.m]: The space is unnecessary, let's revert this

**Comment by NSExceptional** [comment]
@geor-kasapidi bump?

**Comment by geor-kasapidi** [comment]
oh, sorry, i forgot about this pr :) i will update it today

**Comment by NSExceptional** [comment]
Awesome! Don't forget to rebase!

**Pull request closed without merging**